chia/data_layer/s3_plugin_service.py

```diff
@@ -17,6 +20,8 @@
 from chia.types.blockchain_format.sized_bytes import bytes32
 
 log = logging.getLogger(__name__)
+plugin_id = "2bdf9ed6-f3bc-44f6-a69d-06602c0688d9"
```

Comment:
if we want an id + a version we might want to set the id in the config on first instantiation or something so different instances would have different ids